src/database/migrations/0015-alter-bulk_uploads-source_key.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE bulk_uploads
+  DROP COLUMN source_url;
+
+ALTER TABLE bulk_uploads
+  ADD COLUMN source_key VARCHAR NOT NULL;
```

Comment:
Actually, both of these concerns might be addressed by a `RENAME COLUMN` instead of `DROP...ADD COLUMN`! The `source_url` was already of the correct type, and this should only be relevant in dev environments:

```suggestion
ALTER TABLE bulk_uploads
  RENAME COLUMN source_url TO source_key;
```

Reply:
oh much better (and doesn't kill existing data)